### DISCUSSION TITLE
New Rivalry page!

### DISCUSSION THREAD

**Discussion by nmelhado** [nmelhado/league-page#203] [Announcements]
In version 2.2.0, I've introduced a new rivalry page. The rivalry page lets you view the matchup history between two managers, as well as their trade history, and a performance comparison. Any recommendations or comments are welcome!

![localhost_5173_](https://user-images.githubusercontent.com/39418000/232584767-4de8ae08-b072-44bc-947f-3e5d78b5f275.png)

![localhost_5173_ (1)](https://user-images.githubusercontent.com/39418000/232584727-d3c20efb-4d28-4330-bab7-980a3e1ce9ac.png)

_As always, if you've enjoyed League Page, please consider donating. A lot of work goes into maintaining and improving League Page and every donation is very much appreciated!_

**Comment by dawghouse73** [comment]
Love this  page. As a start up, I can see the value in this for years to come. 

I do have one question - the drop down is filled with Sleeper IDs - where can I modify this to show either the current team name (ex. The Love Hurts) for that User ID or their actual name (ex. John - since their names won't change year to year)?

Thanks again...can't brag enough to other leagues about this!

**Reply by nmelhado** [reply]
This is a tough one. It's setup as users not rosters so I can't reliably map to a team. This was intentional so that if managers change or if someone steps in to co-manage, then the stats will correctly track the rivalry. I can't get the users actual name from the Sleeper API either (probably for privacy/security reasons). What I could look into doing is using the manager array to give the users' actual names instead.

I have a few bugs I need to sort through, and plan to address them before the start of the season. I'll try to add this to the list as well.

Would you mind making a feature request (so I don't forget)?

**Comment by dawghouse73** [comment]
Sounds good. I was going to suggest using the managers array from the leagueinfo.js page. I’ll send in a feature request for sure. Keep up the awesome work. Appreciate every second. Jason-----------Jason HouseOn Jun 17, 2023, at 4:10 AM, Nicholas Melhado ***@***.***> wrote:﻿
This is a tough one. It's setup as users not rosters so I can't reliably map to a team. This was intentional so that if managers change or if someone steps in to co-manage, then the stats will correctly track the rivalry. I can't get the users actual name from the Sleeper API either (probably for privacy/security reasons). What I could look into doing is using the manager array to give the users' actual names instead.
I have a few bugs I need to sort through, and plan to address them before the start of the season. I'll try to add this to the list as well.
Would you mind making a feature request (so I don't forget)?

**Comment by sethtobrien** [comment]
Any idea why the rivalry page info wouldn't load? 
![Screenshot 2023-09-12 160337](https://github.com/nmelhado/league-page/assets/140202779/61a35f4a-6bc8-4667-957b-601521074c60)

**Reply by nmelhado** [reply]
I'm not sure. When I visit the same link, it works. Might be a caching issue? Try clearing your cache and seeing if it loads then.
